testsuite/MDAnalysisTests/core/test_atomselections.py

```diff
@@ -516,6 +516,32 @@ def test_molnum(self, universe, selection_string, reference):
         assert_equal(sel.ids, np.array(reference, dtype=np.int32))
 
 
+class TestSelectionRDKit(object):
+    def setup_class(self):
+        pytest.importorskip("rdkit.Chem")
+
+    @pytest.fixture
+    def u(self):
+        smi = "Cc1cNcc1"
+        u = MDAnalysis.Universe.from_smiles(smi, addHs=False,
+                                            generate_coordinates=False)
+        return u
+        
+    def test_aromatic(self, u):
```

Comment:
I think the following 4 tests can be squashed into one just pytest.mark.parametrize